lms/static/js/edxnotes/plugins/accessibility.js

```diff
@@ -148,6 +148,12 @@ define(['jquery', 'underscore', 'annotator'], function ($, _, Annotator) {
                 case KEY.ESCAPE:
                     this.annotator.viewer.hide();
                     break;
+                // Gives focus to the show/hide notes button
+                case KEY.n:
+                    if (event.ctrlKey && event.altKey)  {
+                        $('.action-toggle-notes').focus();
```

Comment:
What if we will simply `disable/enable` notes on `Ctrl + Option/Alt + n` without moving the focus to `.action-toggle-notes` element?
@cptvitamin what do you think?